test/acceptance/agent_configs/minimum_config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "agent": {
+    "run_as_user": "cwagent"
+  },
+  "metrics": {
+    "metrics_collected": {
+      "mem": {
+        "measurement": [
+          "mem_used_percent"
+        ]
```

Comment:
Beyond just configs, consider that installing the agent using the control script also also allows a user to download the configuration file from SSM.
It might be valuable to test that the control script puts the downloaded JSON config in the right location, with the right permissions and owner.
(If it is outside the scope of this test, consider opening another ticket to track that work.)